components/dm/spec/alertmanager.go

```diff
@@ -107,16 +108,23 @@ func (i *AlertManagerInstance) Deploy(t *task.Builder, srcPath string, deployDir
 }
 
 // InitConfig implement Instance interface
-func (i *AlertManagerInstance) InitConfig(e executor.Executor, clusterName, clusterVersion, deployUser string, paths meta.DirPaths) error {
+func (i *AlertManagerInstance) InitConfig(
+	e executor.Executor,
+	clusterName,
+	clusterVersion,
+	deployUser string,
+	paths meta.DirPaths,
+	enableTLS bool,
```

Comment:
Why do we need the `enableTLS` parameter? We can retrieve it from `i.topo` directly.